backend/charts/api/templates/api-env-vars.yaml

```diff
@@ -101,7 +101,9 @@ stringData:
     {{- end }}
 
     NEOSYNC_CLOUD: {{ .Values.neosyncCloud.enabled | default "false" | quote }}
+    {{- if .Values.neosyncCloud.enabled }}
     NEOSYNC_CLOUD_ALLOWED_WORKER_API_KEYS: {{ join "," .Values.neosyncCloud.workerApiKeys }}
+    {{- end }}
```

Comment:
This is fine, for what it's worth, this value is only checked in the code if the above is true. Definitely not clear from looking at the helm chart though.

```
func getAllowedWorkerApiKeys(isNeosyncCloud bool) []string {
	if isNeosyncCloud {
		return viper.GetStringSlice("NEOSYNC_CLOUD_ALLOWED_WORKER_API_KEYS")
	}
	return []string{}
}
```

Reply:
Oh, I see! Thanks for clarifying. The only reason I made this change was because empty strings aren't allowed in Kubernetes secret stringData, so a value must be present. I was getting an error omitting `workerApiKeys` from my values.yaml file.

Reply:
ah! totally makes sense.